apps/angular/src/shared/components/image-file-uploading/image-file-uploading.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="image-file-uploading">
+    <mat-form-field class="image-file-uploading__file-name">
+        <mat-label>Image file</mat-label>
+        <input matInput disabled="true" [value]="getImageName(imageFile$ | async)"/>
+    </mat-form-field>
+    <div>
+        <input hidden type="file" #imageFileInput (change)="uploadImageFile($event)" accept=".jpg, .png"/>    
+        <button mat-icon-button type="button" (click)="imageFileInput.click()" [disabled]="isDisabled">
+            <mat-icon>image</mat-icon>
+        </button>    
+    </div>    
+</div>
+<div class="image-file-preview">
+    <img class="image-file-preview__image" *ngIf="imageFile$ | async as imageFile; else noImageAvailable" [src]="getImageSource(imageFile)" alt=""/>
```

Comment:
```suggestion
    <img 
      *ngIf="imageFile$ | async as imageFile; else noImageAvailable"
       class="image-file-preview__image" 
       [src]="getImageSource(imageFile)" 
       alt=""
     />
```

I've written about this before, try to use structural attributes first. 

For example:
```html
<tag
  *ngIf <- 1st
  class <- 2nd
  etc...
/>
```

Reply:
Sorry, I forgot to check it in other components